docs/DeveloperGuide.adoc

```diff
@@ -422,46 +422,75 @@ image::TimeTableSequenceDiagram.png[width="800"]
+
+Export:
+
+* FreeTime (you export and then import it back in)
+
+* Google Calendar (Preliminary support: TimeSlots show up as individual events for 1 week.)
+
 ==== Current implementation
 
-The import timetable feature allows users to import their timetables from NUSMODS, as an .ics file.
+The import and export feature consists of two parts: +
 
-The timetable feature consists of two parts: +
-1. The backend, which handles the reading of the .ics file and the creation of the timetable object. +
+1. The backend, which handles the reading and writing of _.ics_ files, and the conversion between the .ics format and the TimeTable object. +
 2. The frontend, which parses user input, and informs the user of success or failure.
 
 
-===== Backend implementation
+===== Backend implementation - Import
 
-The file is read and parsed into a TimeTable by IcsUtil class.
+For import, the file is read from the disk using biweekly (3rd-party library), and then subsequently converted into a TimeTable by the IcsUtil class.
 
 The following sequence diagram shows how the TimeTable is imported: +
 
-.Interactions between the Logic Component and Model Component for the `import` Command
-image::ImportSequenceDiagram.png[width="800"]
+.Interactions between the Logic Component and IcsUtil for the `import` Command
+image::ImportSequenceDiagram.png[ ]
+
+===== Backend implementation - Export
+
+For export, the TimeTable is converted into a iCalendar object by IcsUtil, and then written to disk as _.ics_ file using biweekly (3rd-party library).
+
+The following sequence diagram shows how the TimeTable is exported: +
+
+.Interactions between the Logic Component and IcsUtil for the `export` Command
+image::ImportSequenceDiagrsssam.png[width = 600]
 
 {nbsp}
 
+===== Backend implementation - IcsUtil Class
+
 The `IcsUtil` class implements two key utility methods for handling .ics files: +
+
 1. `IcsUtil#readTimeTableFromFile()` - to read an `Optional<TimeTable>` from an .ics file; accepts the `filePath` of the .ics file as parameter +
 2. `IcsUtil#saveTimeTableToFile()` - to write a `TimeTable` to disk; accepts the `TimeTable` and `filePath` of the .ics file as parameter +
 
 
 ===== Frontend implementation
-Same as the other commands, see <<Events-Driven-Nature,*`Events-Driven nature of the design`*>>.
+Similar to the other commands; see <<Events-Driven-Nature,*`Events-Driven nature of the design`*>>.
 
 ==== Design Considerations
 ===== Aspect: Which Classes (and which system(s)) should parse the ics file format?
-* **Alternative 1 (current choice):** All code related to parsing .ics timetable files are inside 'IcsUtil' class.
-** Pros: All functions that have to do with .ics parsing is in 1 class.
+* **Alternative 1 (current choice):** All code related to parsing .ics timetable files are inside 'IcsUtil' class. When executing ImportCommand/ExportCommand, it directly calls a method in IcsUtil.
+** Pros: All functions that have to do with .ics parsing is in one class.
```

Comment:
methods*

Reply:
English... I dunno which one is more grammatic lol

Reply:
As in, it's called "method" in Java but "function" in C and C++

Reply:
like, my thought process is:

When executing ImportCommand or ExportCommand, _it_ directly calls X.

Because they are individual commands; thus singular.

We need Ms Anita